routes/base.php

```diff
@@ -69,7 +69,7 @@
 
     Route::put('/totp', 'SecurityController@generateTotp')->name('account.security.totp');
 
-    Route::post('/totp', 'SecurityController@setTotp');
+    Route::post('/totp', 'SecurityController@setTotp')->name('account.security.totp.set');
```

Comment:
Any reason we need to change the route names? I generally keep them the same if the get/post routes are the same.

Reply:
Without that explicitly, the route doesn't have a name, if I remove it, then this is what I get:

![image](https://user-images.githubusercontent.com/1296882/30755940-6db75688-9f96-11e7-9d3c-a672eae31aad.png)

Which means that I can't reference it via any route name so that I can ignore it in the middleware.

Reply:
Ah, this is fine then for this PR. I think we could make an issue to update all of the route names down the road to be a little more clear. Something using the request type potentiall, not sure if `delete.account.api`/`post.account.api` would make sense, or like `account.delete.api.revoke`?